source/Calamari/Kubernetes/ResourceStatus/ResourceStatusReportExecutor.cs

```diff
@@ -59,7 +59,6 @@ public void ReportStatus(string workingDirectory, ICommandLineRunner commandLine
             if (!definedResources.Any())
             {
                 log.Verbose("No defined resources are found, skipping resource status check");
-                return;
```

Comment:
why did we remove this ?

Reply:
Sorry I forgot to remove update this in the back-and-forth changes regarding the return value. Fixed